server/src/services/cars/addCar.ts

```diff
@@ -0,0 +1,13 @@
+import { Car } from '../../db/models';
+
+const addCarService = async (body, customerId) => {
+  const carRequest = await Car.bulkCreate(
```

Comment:
bulk create used to insert more than a row to db, But we're not doing that right now,
check `Model.create`